### DISCUSSION TITLE
zio-web project status

### DISCUSSION THREAD

**Discussion by timgent** [zio/zio#4880] [Q&A]
Hi! I see [zio-web](https://github.com/zio/zio-web) is listed on the [ecosystem page](https://zio.dev/docs/ecosystem/ecosystem) however doesn't look like there's any published artifacts for the project yet? Just wondered how it was getting along and what plans there are there?

**Comment by adamgfraser** [comment]
@timgent The project is under active development. You can find a video with an update on the latest status here:

https://www.youtube.com/watch?v=UBT-7h8JgU4&feature=youtu.be

**Reply by timgent** [reply]
Thanks @adamgfraser! Are there any plans to publish the library to Maven Central in its current form? I'd love to try using it in it's current state
